src/server/scripts/Commands/cs_account.cpp

```diff
@@ -900,6 +901,60 @@ class account_commandscript : public CommandScript
         }
         return true;
     }
+
+    /// Set email for account
+    static bool HandleAccountSetEmailCommand(ChatHandler* handler, char const* args)
```

Comment:
Use Tail args

Reply:
So we dont need to use any char stuff

Reply:
@Kitzunu i followed the same pattern for the rest of the cs_account.cpp methods, I'm unfamiliar with Tail args?

Reply:
It's the new command implementation. You can see the use of it in other files

Reply:
where can i find documentation on Tail?

Reply:
the only example found is in the cs_go.ccp and it appears you still have to parse it out just the same, not sure what value this would provide?
```
    static bool HandleGoXYZCommand(ChatHandler* handler, Tail args)
    {
        std::wstring wInputCoords;
        if (!Utf8toWStr(args, wInputCoords))
        {
            return false;
        }

        // extract float and integer values from the input
        std::vector<float> locationValues;
        std::wregex floatRegex(L"(-?\\d+(?:\\.\\d+)?)");
        std::wsregex_iterator floatRegexIterator(wInputCoords.begin(), wInputCoords.end(), floatRegex);
        std::wsregex_iterator end;
        while (floatRegexIterator != end)
        {
            std::wsmatch match = *floatRegexIterator;
            std::wstring matchStr = match.str();

            // try to convert the match to a float
            try
            {
                locationValues.push_back(std::stof(matchStr));
            }
            // if the match is not a float, do not add it to the vector
            catch (std::invalid_argument const&){}

            ++floatRegexIterator;
        }
```
Vs. the char const* approach--leaving it below would also maintain the pattern in this file. 
```
    static bool HandleAccountSetEmailCommand(ChatHandler* handler, char const* args)
    {
        if (!*args)
            return false;

        ///- Get the command line arguments
        char* account = strtok((char*)args, " ");
        char* email = strtok(nullptr, " ");
        char* emailConfirmation = strtok(nullptr, " ");
 ```
 
 If there is some other benefit to using Tail vs. char const* I think we should update the documentation first so it is used properly; however, I don't think that should be part of this PR.

Reply:
All scripts should update to the new command script style, and especially new scripts.

But Tail was wrong on my part as it groups everything. Sorry :D (Can I blame it on that it was late at night?)

What we would do is 
```cpp
static bool HandleAccountSetEmailCommand(ChatHandler* handler, AccountIdentifier account, std::string email, std::string emailConfirmation)
```

See cs_character.cpp

Reply:
AccountIdentifier allows you to pass both account name and id into the command

Reply:
ahh sweet thank you so much, the named parameters is a much better approach than the whole string